include/seqan3/range/view/deep.hpp

```diff
@@ -111,6 +111,13 @@ class deep : public detail::adaptor_base<deep<underlying_adaptor_t>, underlying_
     /*!\name Constructors, destructor and assignment
      * \{
      */
+    constexpr deep()                         noexcept = default; //!< Defaulted.
```

Comment:
aren't those inherited by `using base_type::base_type` ?

Reply:
Inherited copy constructors are not considered for copy construction of the derived type. Only for construction of the derived type from the base type. For the default constructors it is not as important, but we have rule-of-six so I added all of them.